UKHomeOffice/sgmr-service
# SPCR-486 Stop PC with charterOrHire null erroring when on edit>submit

### AC / Description of changes
We weren’t translating the response from the API `charterOrHire: null` back to `charterOrHire: 'neitherCharterOrHire'` before prefiling the form in edit scenario

### To Test

- create a pleasure craft with ‘neither’ selected for charter or hire question
- edit that pleasure craft
- change a field other than charter or hire
- save it
- > it should save without error

### Notes

---

**Remember to merge to feature branches not `master`.**
